docs/reference/rack-parameters.md

```diff
@@ -144,6 +144,21 @@ Set this value to the hostname (or IP address) and port number of an HTTP proxy
 
 For more information, see [HTTP Proxy Configuration](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/http_proxy_config.html) in the AWS docs.
 
+#### Attention
+
+The `HttpProxy`param will not change how the apps access the internet, the traffic won't go through the proxy. Only the instances requests will use the proxy for outbound connections, if you want your apps to use the proxy, you have to configure it on the convox.yml:
```

Comment:
Maybe we should add this in the HttpProxy section itself https://docsv2.convox.com/reference/rack-parameters#httpproxy for better visibility.

Reply:
#nit i think it's missing a space here "`HttpProxy`param will not change..."